src/EnergyPlus/DataViewFactorInformation.hh

```diff
@@ -93,8 +99,8 @@ struct ViewFactorInfoData : BaseGlobalStruct
 
     int NumOfRadiantEnclosures = 0; // Number of radiant enclosures
     int NumOfSolarEnclosures = 0;   // Number of solar enclosures
-    Array1D<DataViewFactorInformation::EnclosureViewFactorInformation> EnclRadInfo;
-    Array1D<DataViewFactorInformation::EnclosureViewFactorInformation> EnclSolInfo;
+    EPVector<DataViewFactorInformation::EnclosureViewFactorInformation> EnclRadInfo;
+    EPVector<DataViewFactorInformation::EnclosureViewFactorInformation> EnclSolInfo;
```

Comment:
I love `struct`/`class` type names that include the word "Data" or "Information". So what you're saying is ... this `struct` contains information? I think that's only allowed in C++20 and we're only C++17.